packages/grid/src/Grid.tsx

```diff
@@ -53,6 +53,7 @@ import {
 import { EventHandlerResultOptions } from './EventHandlerResult';
 import { assertIsDefined } from './errors';
 import ThemeContext from './ThemeContext';
+import { getOrThrow } from '.';
```

Comment:
Should import from the file directly instead of from `'.'`, this causes a circular import